google-cloud-clients/google-cloud-storage/src/test/java/com/google/cloud/storage/BucketInfoTest.java

```diff
@@ -121,6 +122,36 @@
           .setLogging(LOGGING)
           .build();
 
+  private static final BucketInfo BUCKET_INFO_ARCHIVE =
+      BucketInfo.newBuilder("b")
+          .setAcl(ACL)
+          .setEtag(ETAG)
+          .setGeneratedId(GENERATED_ID)
+          .setMetageneration(META_GENERATION)
+          .setOwner(OWNER)
+          .setSelfLink(SELF_LINK)
+          .setCors(CORS)
+          .setCreateTime(CREATE_TIME)
+          .setDefaultAcl(DEFAULT_ACL)
+          .setDeleteRules(DELETE_RULES)
+          .setLifecycleRules(LIFECYCLE_RULES)
+          .setIndexPage(INDEX_PAGE)
+          .setIamConfiguration(IAM_CONFIGURATION)
+          .setNotFoundPage(NOT_FOUND_PAGE)
+          .setLocation(LOCATION)
+          .setLocationType(LOCATION_TYPE)
+          .setStorageClass(ARCHIVE_STORAGE_CLASS)
```

Comment:
Is it typical to set all of these options while creating a bucket in Java? Just curious.

Reply:
No need to set all of these option but yes bucket name is required parameter see the [api doc](https://cloud.google.com/storage/docs/json_api/v1/buckets/insert) and example [create bucket](https://cloud.google.com/storage/docs/creating-buckets#storage-create-bucket-java)

Reply:
Okay, sounds good.